packages/webpack-cli/lib/groups/StatsGroup.js

```diff
@@ -15,14 +15,10 @@ class StatsGroup extends GroupHelper {
     resolveOptions() {
         if (this.args.verbose && this.args.stats) {
             logger.warn('Conflict between "verbose" and "stats" options. Using verbose.');
-            this.opts.option.stats = {
-                verbose: true,
-            };
+            this.opts.options.stats = 'verbose';
```

Comment:
The current behavior is that passing only `--verbose` also invokes `this.args.stats`.
> `this.args` & `this.opts` for `webpack --verbose`

>![Screenshot at 2020-05-16 10-01-20](https://user-images.githubusercontent.com/46647141/82110688-e8b2b500-975d-11ea-8281-07c3f330717f.png)

Looking into it.

Reply:
`this.args.stats` is always true.

![Screenshot at 2020-05-16 14-14-38](https://user-images.githubusercontent.com/46647141/82115309-9edac680-977f-11ea-9cac-9027ddd4eb43.png)